keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/ManualJudgementNotifier.kt

```diff
@@ -18,13 +18,13 @@ import org.springframework.stereotype.Component
 
 @Component
 @Configuration
-@EnableConfigurationProperties(ManualJudgementNotificationConfig::class)
+@EnableConfigurationProperties(KeelNotificationConfig::class)
 /**
```

Comment:
I think we could move this into the new notifier if we wanted manual judgment notifications to repeat more than once. However, I don't think we want that right now, so I think it belongs in a separate Notifer.